src/controller/file/serializing/filter_serialization.py

```diff
@@ -64,16 +64,67 @@ def create_channel_mappings_for_filter_set_for_fish(om: SceneOptimizerModule, sc
     om.wrap_up(scene_element)
     channel_links_to_be_created: list[tuple[Filter, ElementTree.SubElement]] = om.channel_link_list
     override_port_mapping: dict[str, str] = om.channel_override_dict
+    default_nodes: dict[DataType, list] = dict()
+    time_node = None
+    for f_entry in channel_links_to_be_created:
+        if f_entry[0].filter_type == FilterTypeEnumeration.FILTER_TYPE_TIME_INPUT:
+            time_node = f_entry[0].filter_id
+            break
     for f_entry in channel_links_to_be_created:
         filter_ = f_entry[0]
         filter_element = f_entry[1]
         for channel_link in filter_.channel_links.items():
             output_channel_id: str = channel_link[1]
+            if output_channel_id == "":
+                continue
```

Comment:
Is this the way to handle removed connections?
Like here https://github.com/Mission-DMX/Project-Editor/blob/277e766f603bbd630fae29ccc15c3f40223f9cd2/src/view/show_mode/editor/nodeeditor.py#L72-L73,

if a connection gets removed the channel_link stays in filter.channel_links but the output part gets set with ""

Reply:
There is. It's a dictionary and you can remove the item altogether. But where do we need to remove a filter connection instead of modifying it?

Reply:
> There is. It's a dictionary and you can remove the item altogether. But where do we need to remove a filter connection instead of modifying it?

if you delete a connection and do not set it again to another node (to use the default value)